sdk/data/aztables/zc_client_options.go

```diff
@@ -5,15 +5,20 @@ package aztables
 
 import (
 	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
 )
 
 // ClientOptions are the optional parameters for the NewClient method
 type ClientOptions struct {
 	azcore.ClientOptions
 }
 
-func (c *ClientOptions) toPolicyOptions() *azcore.ClientOptions {
-	return &azcore.ClientOptions{
+func (c *ClientOptions) toPolicyOptions() *policy.ClientOptions {
```

Comment:
should we make it possible to disable the tenant discovery behavior through clientOptions?

Reply:
Did you add that for .NET in storage? I don't have it in the KV clients, but if it's included in other languages I can add it in Go